### DISCUSSION TITLE
Data API Builder (DAB)  GraphQL Mutations - Stored Procedures - OUTPUT parameters ?

### DISCUSSION THREAD

**Discussion by esobrino** [Azure/data-api-builder#1841] [Q&A] [answered]
DAB is awesome!

In DAB all is good so far.  I am calling an SP and have an OUTPUT parameter and will like to get the return-value also.

I have not found any documentation on such, let me know if this is possible... or I need to return a result set with those?

Hope to hear from anyone soon,  Ed.

**Comment by seantleonard** [comment, marked as answer]
Hi @esobrino, We don't support output parameters at the point in time. You'll need to make sure the first result set returned by your stored procedure includes the data you want.
